lxd-migrate/lxd.go

```diff
@@ -218,10 +218,6 @@ func (d *lxdDaemon) reload() error {
 		return systemdCtl("reload", "snap.lxd.daemon.service")
 	}
 
-	if osInit() == "upstart" {
```

Comment:
lets just leave this change out as we dont want to be removing stuff from the LTS